server/controllers/userController.js

```diff
@@ -35,9 +35,37 @@ const createUser = async (req, res) => {
   return res.status(200).json(createdUser);
 }
 
-const updateUser = () => {
-  // TODO 2 - Actualizar información del usuario (Rocio)
-}
+const updateUser = (async (req, res) => {
+  try {
+    const { id } = req.params;
+    const { first_name, last_name, email, password, age, photo, phone_number, gender, identity_document, status, birth_date } = req.body;
+    const selectedUser = await user.findByPk(id);
+
+    if (selectedUser === null) {
+      res.status(400)
+      throw new Error('Tarea no encontrada');
+
+    } else {
+      selectedUser.first_name = first_name;
+      selectedUser.last_name = last_name;
+      selectedUser.email = email;
+      selectedUser.password = password;
+      selectedUser.age = age;
+      selectedUser.photo = photo;
+      selectedUser.phone_number = phone_number;
+      selectedUser.gender = gender;
+      selectedUser.identity_document = identity_document;
+      selectedUser.status = status;
+      selectedUser.birth_date = birth_date;
+
+      await selectedUser.save();
+      res.json(selectedUser);
```

Comment:
Es buena práctica que agregues el return siempre antes de mandar las respuestas. Reduces la posibilidad de un error porque el proceso siguió ejecutándose. 

```suggestion
      return res.json(selectedUser);
```